epoch-verifier/src/data/context.ts

```diff
@@ -1,36 +1,43 @@
-import { Address, Allocation, EpochInfo, Reward, UserBudget, UserDonation } from "./models"
+import {
+  Address,
+  Allocation,
+  AllocationRecord,
+  EpochInfo,
+  Reward,
+  UserBudget,
+  UserDonation
+} from "./models"
 
 export interface Context {
   allocations: Allocation[]
-  budgets: Map<Address, bigint> 
+  budgets: Map<Address, bigint>
   epochInfo: EpochInfo
   rewards: Reward[]
 }
 
-export function buildContext(budgets: UserBudget[], allocations: Allocation[], rewards: Reward[], epochInfo: EpochInfo): Context{
+export function buildContext(budgets: UserBudget[], allocations: AllocationRecord[], rewards: Reward[], epochInfo: EpochInfo): Context {
 
-  const filteredBudgets = budgets.filter((ub) => ub.budget !== BigInt(0))
-  const filteredAllocs = allocations.filter(alloc => alloc.donations.length !== 0)
+  const filteredBudgets = budgets.filter((ub) => ub.amount !== BigInt(0))
```

Comment:
`positiveBudgets` instantly let the reader know what it means, whereas "filteredBudges" forces the reader to peak into the function. Iterative element of "budgets" is "budget", not "ub", an abbreviation someone may not be familiar with.

```suggestion
  const positiveBudgets = budgets.filter((budget => budget.amount !== BigInt(0))
```

Reply:
Done.